### ISSUE TITLE
PM Install-Package error

### ISSUE THREAD

**Issue by jpeltonen** [3F/DllExport#32]
Tried to install in Visual Studio 2013 and got Import-Module error.

![dllexport_install_error](https://cloud.githubusercontent.com/assets/22810621/25529824/68cf1940-2c2c-11e7-9b2a-f29b9e430964.png)

**Comment by 3F** [comment]
Can you check with v1.5.1 / v1.4.0 ?

```powershell
Install-Package DllExport -Version 1.5.1
```

**Comment by 3F** [comment]
also I see that v1.5.2 now gives with VS2015 (I remember exactly that it was OK before o_o) the following error when Install -> Remove -> Install (Cecil & Direct-Mod):

```bash
binmod: lpos(1425); ident(32)
binmod: buffer(500); fullseq(536)
```

```powershell
Executing script file 'D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\tools\init.ps1'...
The names of some imported commands from the module 'DllExportCmdLets' include unapproved verbs that might make them less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose parameter. For a list of approved verbs, type Get-Verb.
Executing script file 'D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\tools\install.ps1'...
set new namespace(Cecil: True): (DllExportTSLib2) - (D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\lib\net20\DllExport.dll)
cecil: NS property has been updated for DllExportAttribute.

The DllExport Library has been modified !

namespace: 'DllExportTSLib2' :: D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\lib\net20\DllExport.dll
Details here: https://github.com/3F/DllExport/issues/2
The Export has been configured for platform: x86
The Base for ordinals: 1
Generate .exp + .lib via MS Library Manager: False
Use our IL Assembler: True
Successfully installed 'DllExport 1.5.2' to DllExportTSLib2
```
v
```powershell
Executing script file 'D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\tools\uninstall.ps1'...
Removed package 'DllExport.1.5.2' from 'packages.config'
Successfully uninstalled 'DllExport.1.5.2' from DllExportTSLib2
Removing package 'DllExport.1.5.2' from folder 'D:\tmp\2015\DllExportTSLib2\packages'
Skipping 'D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\lib/net20/DllExport.dll' because it was modified.
Removed package 'DllExport.1.5.2' from folder 'D:\tmp\2015\DllExportTSLib2\packages'
```
v
```powershell
Executing script file 'D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\tools\install.ps1'...
Set-Configuration : Cannot bind parameter 'MetaLib' to the target. Exception setting "MetaLib": "[A]net.r_eg.DllExport.
Configurator.SetConfigurationCmdlet cannot be cast to [B]net.r_eg.DllExport.Configurator.SetConfigurationCmdlet. Type A
 originates from 'net.r_eg.DllExport.Configurator, Version=1.0.0.0, Culture=neutral, PublicKeyToken=e25d2e78e2441842' i
n the context 'LoadNeither' in a byte array. Type B originates from 'net.r_eg.DllExport.Configurator, Version=1.0.0.0, 
Culture=neutral, PublicKeyToken=e25d2e78e2441842' in the context 'LoadNeither' in a byte array."
At D:\tmp\2015\DllExportTSLib2\packages\DllExport.1.5.2\tools\install.ps1:22 char:28
+ Set-Configuration -MetaLib "$metaLib" -InstallPath "$installPath" -To ...
+                            ~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Set-Configuration], ParameterBindingException
    + FullyQualifiedErrorId : ParameterBindingFailed,net.r_eg.DllExport.Configurator.SetConfigurationCmdlet
```

**Comment by 3F** [comment]
I closed this issue because of new [Wizard for v1.6+ versions](https://github.com/3F/DllExport/issues/38). You already can test it via our beta releases:

* Latest manager: https://3F.github.io/DllExport/releases/latest/manager/
* Please note again - you shouldn't use standard nuget clients anymore: https://www.youtube.com/watch?v=okPThdWDZMM 

If you still have a some related problems, please create new issue for new planned Wizard.
